projects/observability/src/shared/components/topology/d3/d3-topology.ts

```diff
@@ -159,8 +164,10 @@ export class D3Topology implements Topology {
   }
 
   private updateLayout(): void {
+    this.runAndDrainCallbacks(this.dataClearCallbacks);
+    this.collapseGroupNodesIfPresent();
```

Comment:
can we redraw only if group node is present? Otherwise, we can continue to update the positions of already rendered nodes.

Reply:
Done